Shell/main.cpp

```diff
@@ -5,13 +5,15 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <dirent.h>
```

Comment:
Do we actually need dirent.h?

Reply:
Right, we don't now I use CDirIterator ;)